engine/src/main/java/org/destinationsol/game/ship/hulls/Hull.java

```diff
@@ -57,6 +58,7 @@
     private float myAngle;
     private float myRotSpd;
     private ShipEngine myEngine;
+    private List<ParticleEmitter> myParticleEmitters;
```

Comment:
`particleEmitters`. Again, I realize all other fields are `my*`, but let's keep the new code stink free.